src/main/resources/archetype-resources/Jenkinsfile

```diff
@@ -2,4 +2,4 @@
 
 library('jenkins-libraries@v3.2.1')
 
-springBoot2Pipeline(jdk: 'jdk17', email: '${mailerRecipients}')
+springBoot2Pipeline(jdk: 'jdk17', qualityGateCheck: true, email: '${mailerRecipients}')
```

Comment:
Not sure why we didn't set it as default in the Jenkins job, but here

Reply:
This property enables build failure when Sonar quality gate isn't passed. Most of our silos currently don't have proper configuration of sonar and have quality gate checks failed. So assigning this property here will affect only new silos that going to be created from this archetype without affecting old ones.

Reply:
:+1: